razorpay-payments.php

```diff
@@ -527,14 +572,19 @@ function check_razorpay_response()
             {
                 $success = false;
                 $error = 'Customer cancelled the payment';
-
                 $this->handleErrorCase($order);
             }
 
             $this->updateOrder($order, $success, $error, $razorpayPaymentId);
 
+            $this->redirectUser($order);
+        }
+
+        protected function redirectUser($order)
+        {
```

Comment:
@captn3m0 You don't. 
I moved 
`$this->add_notice($this->msg['message'], $this->msg['class']);`
to 
`public function updateOrder(& $order, $success, $errorMessage, $razorpayPaymentId)`